gravityflow/gravityflow
# Fix File Upload Validation Issue on User Input Step

## Description
Closes [Issue#90](https://github.com/gravityflow/backlog/issues/90)
File Upload field generates a Validation Error on User Input Step for Save Progress

## Testing instructions
1. Create a form with a text field and a hidden File Upload field. The File Upload field should be marked 'Required'
2. Create a User Input Workflow Step with the File Upload field and the text field as the Editable fields. Also, enable the 'Save Progress' option on the step.
3. Process an entry on the form.
4. Check the workflow inbox for the entry.
5. Update the text field input, without doing anything on File Upload (let it remain blank).
6. 'Save Progress' for the workflow step.
7. Validation error occurs.

## Automated Test Enhancements
Added test `0054FileUploadValidationCept.php` to check for validation error of File Upload field on User Input Step.
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
